announcements/release-4.1.html

```diff
@@ -0,0 +1,195 @@
+		It is part of the Astropy Project, which aims to foster an ecosystem of
+		interoperable astronomy packages for Python.
+		</p>
+		<p>
+		New and improved major functionality in this release includes:
+		<ul>
+			<li>A new SpectralCoord class for representing and transforming spectral quantities</li>
+			<li>Support for writing Dask arrays to FITS files</li>
+			<li>Added True Equator Mean Equinox (TEME) frame for satellite two-line ephemeris data</li>
+			<li>Support for in-place setting of array-valued SkyCoord and frame objects</li>
+			<li>Change in the definition of equality comparison for coordinate classes</li>
+			<li>Support use of SkyCoord in table vstack, dstack, and insert_row</li>
+			<li>Support for table cross-match join with SkyCoord or N-d columns</li>
+			<li>Support for custom attributes in Table subclasses</li>
+			<li>Added a new Time subformat unix_tai</li>
+			<li>Added support for the -TAB convention in FITS WCS</li>
+			<li>Support for replacing submodels in CompoundModel</li>
+			<li>Support for units on otherwise unitless models via the Model.coerce_units method.</li>
+			<li>Support for ASDF serialization of models</li>
+		</ul>
+		</p>
+		<p>
+		In addition, hundreds of smaller improvements and fixes have been made. An overview of the changes is provided at:
+		</p>
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://docs.astropy.org/en/stable/whatsnew/4.1.html">http://docs.astropy.org/en/stable/whatsnew/4.1.html</a>
+		</p>
+		<p>
+		Instructions for installing Astropy are provided on our <a
+		href="http://www.astropy.org">website</a>, and extensive documentation can be
+		found at:
+		</p>
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://docs.astropy.org">http://docs.astropy.org</a>
+		</p>
+		<p>
+		  If you usually use pip/vanilla Python, you can do:
+		</p>
+<pre>
+pip install astropy --upgrade
+</pre>
+		<p>
+		If you make use of the <a href="https://www.continuum.io/downloads">Anaconda
+		Python Distribution</a>, soon you will be able update to Astropy v4.1 with:
+		</p>
+<pre>
+conda update astropy
+</pre>
+		<p>
+		Or if you can't wait for Anaconda to update their default version you can use the astropy channel:
+		</p>
+<pre>
+conda update -c astropy astropy
+</pre>
+		<p>
+		Please report any issues, or request new features via our GitHub repository:
+		</p>
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/astropy/astropy/issues">https://github.com/astropy/astropy/issues</a>
+		</p>
+		<p>
+		Nearly 400 developers have contributed code to Astropy so far, and you can find out more about the team behind Astropy here:
+		</p>
+		<p>
+		&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://www.astropy.org/team.html">http://www.astropy.org/team.html</a>
+		</p>
+		<p>
+			The LTS (Long Term Support) version of Astropy at the time of v4.1's release is v4.0 - this version will be maintained until next LTS release (v5.0, scheduled for fall 2021).
```

Comment:
I guess season defined in northern hemisphere without explanation is a given?
```suggestion
			The LTS (Long Term Support) version of Astropy at the time of v4.1's release is v4.0 - this version will be maintained until next LTS release (v5.0, scheduled for Fall 2021).
```